nuxt-frontend/components/android-app-development/SuccessStorySection.vue

```diff
@@ -35,7 +37,7 @@
                 class="mb-8 mx-auto rounded-[5px] sm:px-10 md:px-6 lg:px-4 xl:px-24 p-5 bg-[#F2709C]/[0.08]"
```

Comment:
this background is used at many places so, can you define it in the config file with the required opacities